gaspy/atoms_operators.py

```diff
@@ -251,6 +252,172 @@ def find_adsorption_sites(atoms):
     sites = sites_dict['all']
     return sites
 
+def find_bulk_cn_dict(bulk_atoms):
+    '''
+    Get a dictionary of coordination numbers
+    for each distinct site in the bulk structure.
+
+    Taken from pymatgen.core.surface Class Slab
+    `get_surface_sites`.
+    https://pymatgen.org/pymatgen.core.surface.html
+    '''
+    struct = AseAtomsAdaptor.get_structure(bulk_atoms)
+    sga = SpacegroupAnalyzer(struct)
+    sym_struct = sga.get_symmetrized_structure()
+    unique_indices = [equ[0] for equ in sym_struct.equivalent_indices]
+    # Get a dictionary of unique coordination numbers
+    # for atoms in each structure.
+    # for example, Pt[1,1,1] would have cn=3 and cn=12
+    # depends on the Pt atom.
+    v = VoronoiNN()
```

Comment:
Single-letter object names are not a good practice. Expand it out to be more descriptive.